### PR TITLE
Front-end model changes

### PR DESCRIPTION
**Pull request by JeffreyDevloo**

### CODE REVIEW

File: webapps/api/backend/serializers/serializers.py [hunk 1]
```diff
@@ -17,57 +17,270 @@
...
+        Specifying it 2 with _relations_contents given will serialize the relations of the fetched relation. This causes a chain of serializations
+        - dynamic_property_1,dynamic_property_2 (results in static properties plus 2 dynamic properties)
+        Properties can also be excluded by prefixing the field with '-':
+        - contents=_dynamic,-dynamic_property_2,_relations (static properties, all dynamic properties except for dynamic_property_2 plus all relations)
+        Relation serialization can be done by asking for it:
+        - contents=_relations,_relations_contents=re-use
+        :type contents: list or str
+        :raises UnsupportedContentException: If a content string is passed which is not valid
+        """
+        super(ContentOptions, self).__init__()
```
Comment (sjovdnbos) [inline webapps/api/backend/serializers/serializers.py]: super call but no super class. Calling object init? For what purpose?

**Review by sjovdnbos — APPROVED** [review]

File: webapps/api/backend/serializers/serializers.py (hunk 1 above)
Comment (JeffreyDevloo) [reply webapps/api/backend/serializers/serializers.py]: Following the inheritance model properly :D

**Comment by JeffreyDevloo** [comment]
Requires adjustment to the alba-plugin and iscsi-plugin.
Mainly used in the Dual Controller logic

**Comment by JeffreyDevloo** [comment]
Depends on Dual Controller related changes in plugins to be ready too

**Pull request merged**